instagram/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import Comment from './Comment';
+import moment from 'moment';
+import styled from 'styled-components';
+
+const CommentSectionWrapper = styled.div `
```

Comment:
personally I prefer to put the styling at the bottom of the page, you could even export them from a styling file similar to what you would do for a css file if you wanted to. My reasoning is to keep the focus on the main component that you have created, chances as that when you come into a file, you're not coming in to look at the styling often.. it's usually to look at some code or refactor it. With this in mind, I would say to try and put the styling last on your file.